companies/dbrs-morningstar.json

```diff
@@ -0,0 +1,28 @@
+{
+    "slug": "dbrs-morningstar",
+    "relevant-countries": [
+        "us",
+        "ca",
+        "fr",
+        "it",
+        "es",
+        "dk"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "DBRS Morningstar",
+    "address": "DBRS Limited, DBRS Tower 181 University Avenue, Suite 700, Toronto, ON  M5H 3M7",
```

Comment:
The [privacy policy](https://www.dbrs.com/privacy-policy/) lists a number of companies under `I.` — these should be listed under the `runs` key.